conversions.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto2";
+
+package cakeproto;
+
+import "common.proto";
+
+message Conversion {
+  optional int64 id = 1;
+  optional int64 visitor_id = 2;
+  optional int64 request_session_id = 3;
+  optional int64 click_id = 4;
+  optional int64 click_request_session_id = 5;
+  optional TimeWithZone conversion_date = 6;
+  optional TimeWithZone click_date = 7;
+  optional TimeWithZone last_updated = 8;
+
+  optional IdName event = 9;
+  optional IdName affiliate = 10;
+  optional IdName advertiser = 11;
+  optional IdName offer = 12;
+  optional IdName offer_contract = 13;
+  optional int64 campaign_id = 14;
+  optional IdName campaign_type = 15;
+  optional IdName creative = 16;
+
+  optional string sub_id1 = 17;
+  optional string sub_id2 = 18;
+  optional string sub_id3 = 19;
+  optional string sub_id4 = 20;
+  optional string sub_id5 = 21;
+
+  optional string ip_address = 22;
+  optional string click_ip_address = 36;
+  optional string user_agent = 23;
+  optional string click_user_agent = 37;
+  optional string source_type = 24;
+
+  optional IdName price_format = 25;
+  // paid_unbilled and received_unbilled omitted - not available in ConversionChanges API
+  optional MoneyWithCurrency paid = 26;
+  optional MoneyWithCurrency received = 27;
+
+  optional int64 step_reached = 28;
+  // pixel_dropped, suppressed and returned omitted - not reproducible in UI
+  // test omitted - fetch only non-test conversions
+  optional string transaction_id = 29;
+  optional Disposition current_disposition = 30;
+  // order_total omitted - not reproducible in UI
```

Comment:
Okay, I will look into `order_total`.

It can mean both. Sometimes cake doesn't return the value at all, sometimes it's nil. Probably depends on which of theirs devs wrote the API code ;)